### PR TITLE
update buffer feature

### PR DESCRIPTION
**Pull request by Mars12-dev**
I updated the function setBuffer and setTotalSupply in the contract TapETH.

### CODE REVIEW

**Review by ukby1234 — APPROVED** [review]

File: contracts/TapETH.sol [hunk 1]
```diff
@@ -271,10 +273,8 @@ contract TapETH is Initializable, ITapETH {
   function setTotalSupply(uint256 _amount) external {
     require(pools[msg.sender], "TapETH: no pool");
     require(_amount != 0, "TapETH: no pool");
-    uint256 _deltaBuffer = Math.min(buffer, _amount);
+    uint256 _deltaBuffer = (buffer * _amount) / BUFFER_DENOMINATOR;
     _totalSupply += _amount - _deltaBuffer;
-    totalRewards += _amount;
```
Comment (zjb0807) [inline contracts/TapETH.sol:276]: `totalRewards` is only used here, do you want to remove it?

Comment (Mars12-dev) [reply contracts/TapETH.sol:276]: yes I thought that I have already removed

Comment (zjb0807) [reply contracts/TapETH.sol:276]: I think we need the `totalRewards` to record the rewards. 
@ukby1234 What do you think?

Comment (ukby1234) [reply contracts/TapETH.sol:276]: not sure though

Comment (zjb0807) [reply contracts/TapETH.sol:276]: This is just a rewards record and has no logical processing. But I suggest leaving it

**Pull request merged**